ZuluContent/Mobiles/BaseCreature/BaseCreature.cs

```diff
@@ -1867,6 +1868,13 @@ public bool Tamable
             set { m_bTamable = value; }
         }
 
+        [CommandProperty(AccessLevel.GameMaster)]
+        public DateTime UnresponsiveToTamingEndTime
```

Comment:
I think the recommended way for timers/endtimes etc is to use the `Core.TickCount` apparently it's a lot more efficient than setting datetimes and comparing them everywhere.

I try convert them all when I brought over RunUO.T2A to ModernUO but there might still be some vestiges so don't use them as an example!

e.g.

```csharp
// Don't forget about auto properties! You don't need an m_timeUnresponsiveToTaming
public long UnresponsiveToTamingEndTime { get; set; } 
```

```csharp
creature.UnresponsiveToTamingEndTime = Core.TickCount + (int)TimeSpan.FromSeconds(delay).TotalMilliseconds;
```

Reply:
Cool will do